api/callJokeApi.js

```diff
@@ -0,0 +1,50 @@
+const https = require('node:https');
+const { parseData } = require('../utils/supportFunctions')
+
+/**
+ * Function check if term was provided and checks its type
+ * @param {string} term 
+ */
+const validateTerm = (term) => {
```

Comment:
In your opinion, would it make sense to put all the validators in a separate file?

Reply:
I can create a separate file for it. I think if there were more validations it would be necessary, but for such a small project I don't see a burning need